src/null.ts

```diff
@@ -0,0 +1,46 @@
+import { DmmfDocument, DmmfTypes } from './dmmf'
+
+/**
+ * Take the incoming GraphQL args of a resolver and replaces all `null` values
+ * that maps to a non-nullable field in the Prisma Schema, by `undefined` values.
+ * 
+ * In Prisma, a `null` value has a specific meaning for the underlying database.
+ * Therefore, `undefined` is used instead to express the optionality of a field.
+ * 
+ * In GraphQL however, no difference is made between `null` and `undefined`.
+ * This is the reason why we need to convert all `null` values that were assigned to `non-nullable` fields to `undefined`.
+ */
+export function transformNullsToUndefined(
+  args: Record<string, any>,
+  schemaArgs: Record<string, DmmfTypes.SchemaArg>,
+  dmmf: DmmfDocument,
```

Comment:
IIUC `schemaArgs` data is derived from `dmmf`.

Not sure it makes sense in this case, but my inclination would be to have distinctly named and designed dmmf transformations, with sets of functions being tailored to work each one and only one (ideally).

Generic and vague point. 

Might not apply here.

Reply:
Apologies, I think I don't understand your point here

Reply:
The particular point here being that, there are two params `dmmf` and `schemaArgs` which appear slightly related but slightly not. Was wondering if we can get to a point where there is only 1 param / 1 [maybe large] data structure.

Maybe go deeper in a future call/pr, all good!